pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -31,7 +32,7 @@ type Client struct {
 }
 
 // NewClient creates new KUDO Client
-func NewClient(kubeConfigPath string, requestTimeout int64) (*Client, error) {
+func NewClient(kubeConfigPath string, requestTimeout int64, validateInstall bool) (*Client, error) {
```

Comment:
This is also called in `pkg/kudoctl/util/kudo/kudo_test.go` in addition `pkg/kudoctl/env/environment.go`. Please change the call there accordingly.

Reply:
Thanks, fixed